src/components/Pages/HomeSections/whyUs/whyUsModel.ts

```diff
@@ -0,0 +1,13 @@
+import { LegacyRef } from "react";
+
+export interface CardModel{
+    title: string,
+    description: string,
+    img: JSX.Element;
+}
+
+export interface LangType {
+    currentLang: Record<string, string>;
+    inSection: boolean;
```

Comment:
Why not name it "isVisible"? That's how you define it in WhyUsSection.

Reply:
done